tests/access_test.go

```diff
@@ -53,6 +54,57 @@ var _ = Describe("[rfe_id:500][crit:high][vendor:cnv-qe@redhat.com][level:compon
 		tests.BeforeTestCleanup()
 	})
 
+	Describe("With the kubevirt-controller service account", func() {
+		table.DescribeTable("should verify permissions on resources are correct for kubevirt-controller", func(resource string) {
+
+			controllerVerbs := make(map[string]string)
+			controllerVerbs["list"] = "yes"
+			controllerVerbs["watch"] = "yes"
+			controllerVerbs["delete"] = "no"
+			controllerVerbs["update"] = "no"
+			controllerVerbs["deleteCollection"] = "no"
+
+			// Adjustment for the rule associated with kubevirt, vmipresets, vmim
+			switch resource {
+			case "kubevirt":
+				controllerVerbs["get"] = "yes"
+				controllerVerbs["patch"] = "no"
+				controllerVerbs["create"] = "no"
+			case "virtualmachineinstancepresets":
+				controllerVerbs["get"] = "no"
+				controllerVerbs["patch"] = "no"
+				controllerVerbs["create"] = "no"
+			case "virtualmachineinstancemigrations":
+				controllerVerbs["get"] = "yes"
+				controllerVerbs["patch"] = "yes"
+				controllerVerbs["create"] = "yes"
+			default:
+				controllerVerbs["get"] = "yes"
+				controllerVerbs["patch"] = "yes"
+				controllerVerbs["create"] = "no"
+			}
+
+			namespace := tests.NamespaceKubevirt // Is there an exported var defines kubevirt ns ?
```

Comment:
I'm not finding and exported var for this, but I also see this var is not used anywhere. It looks like the pattern is to use `GetCurrentKV()` and then use it's namespace or you could potentially use the `flags.KubeVirtInstallNamespace` var?

Reply:
I changed the `test/utils.go` to export the `tests.NamespaceKubevirt`. I will checkout the pattern around GetCurrentKV() as you suggested.